hessian-lite/src/main/java/com/alibaba/com/caucho/hessian/io/AbstractHessianOutput.java

```diff
@@ -1,3 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
double license head

Reply:
Do we only need to add the apache 2.0 licence to the source file that has no copyright notice?

Reply:
I think so 。 @beiwei30 ，shall we retain the old copyright info？

`/*
   * Copyright (c) 2001-2004 Caucho Technology, Inc.  All rights reserved.
   *`

Reply:
we should keep this.

Reply:
@aspirewit ,  according to what @beiwei30  said

Reply:
Ok, I will add the apache 2.0 licence to the source file that has no copyright notice.